.ci/docker/Dockerfile_dswx_ni

```diff
@@ -38,11 +38,12 @@ LABEL org.label-schema.build-date=${BUILD_DATE_TIME} \
 
 # Copy the OPERA PGE software into the container
 # the base container has a default user "mamba" with UID/GID 1000/1000
-COPY --chown=mamba:mamba ${PGE_SOURCE_DIR} ${PGE_DEST_DIR}
+COPY --chown=dswx_user:dswx_user ${PGE_SOURCE_DIR} ${PGE_DEST_DIR}
 
 # Switch to root for installing into Conda Env
 USER 0:0
 
+SHELL ["conda", "run", "-n", "DSWX-SAR", "/bin/bash", "-c"]
```

Comment:
Maybe add a comment above:
Set the default shell to run commands within the Conda environment named "DSXW-SAR" using bash.